### PR TITLE
[MIRROR] Rat Kebabs

### PR DESCRIPTION
**Pull request by ghost**
Original PR: https://github.com/tgstation/tgstation/pull/32862
--------------------
:cl: Code by Pyko, Ported by Frozenguy5
add: Rat Kebabs and Double Rat Kebabs have been added!
/:cl:

[why]: more food = fun

pyko gave me permission to do this here. original by pyko - https://github.com/HippieStation/HippieStation/pull/4850

### CODE REVIEW

**Comment by Pyko1** [comment]
Removed the rat-kebab/double rat-kebab from our own module, was too lazy to remove the icons from our .dmi tho.

File: hippiestation/code/modules/food_and_drinks/food/snacks_meat.dm [hunk 1]
```diff
@@ -1,19 +1,3 @@
...
-	tastes = list("rat meat" = 1, "metal" = 1)
-	foodtype = MEAT | GROSS
-
-/obj/item/reagent_containers/food/snacks/kebab/rat/double
-	name = "double rat-kebab"
-	icon_state = "doubleratkebab"
-	tastes = list("rat meat" = 2, "metal" = 1)
-	bonus_reagents = list("nutriment" = 6, "vitamin" = 2)
-
 /obj/item/reagent_containers/food/snacks/kebab/butt
```
Comment (JohnGinnane) [inline hippiestation/code/modules/food_and_drinks/food/snacks_meat.dm:1]: Should this not also be removed?

**Review by JohnGinnane — APPROVED** [review]
🌮 💯

File: hippiestation/code/modules/food_and_drinks/food/snacks_meat.dm (hunk 1 above)
Comment (Pyko1) [reply hippiestation/code/modules/food_and_drinks/food/snacks_meat.dm]: naah, tg only added rat and double rat

**Pull request merged**